try{
    let year = 2020;
    alert(year);

    let arrreglo = new Array(999999999999999999)
}catch(error){   //captura el error si hay un fallo en el codigo anterior
   
    console.log(error)
    alert('Error en el codigo')
}